### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_Screenreader:Voice over is not announcing level of the drop down box,edit box and check box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6029]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS4.1.2; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name:Voice Over
MAS Reference: MAS1.3.1,MAS4.1.2
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab from Home page->Click on new job button->Select steps tab.
4.Go to the "New step" button hit enter/spacebar
5.navigate to "Advanced' button and navigate with Tab till output file button.
5.Start Voice over
6.Observed that Voice over is not announcing the level.
**Actual:**
**Issue1**.Voice Over is not announcing the level for the drop down box "On success action" and "on failure action".
**Issue2**.Voice Over is not announcing the level for the edit box "Run as user".
**Issue3**.Voice Over is not announcing the level for the check box "log to table", "Append output 
to existing entry tableland "Include step output in history"
**Issue4.
**Voice Over is not announcing the level for the spinner "Retry Attemps" and "Retry interval(minutes)"

**Similar issue has been observed for below pages** -
**1. CreateAlert -** Go to the SQL server agent tab->On alerts tab->Click on new alert button
**2. CreateAlertResponse -** Go to the SQL server agent tab->On alerts tab->Click on new alert button  -> Select Response tab.
**3. CreateAlertOptions -** Go to the SQL server agent tab->On alerts tab->Click on new alert button -> Select Options tab.
**4. NewJobNotifications -** Navigate to the SQL agent tab from Home page->Click on new job button     ->Select Notifications tab.
5.**CSVModifyColumns**- Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.

**Expected:**
Voice Over should announce the level of the button so that user will understand the purpose of that button.
**User Impact:**
Screen reader user will not understand the level assigned to each process. They will not unserstand that the current level of each process.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
**MAS1.3.1,MAS4.1.2**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
**Attachment for Reference:**
[6029_[Accessibility]A11y_AzureDataStudio_NewJobSteps_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3290585/6029_.Accessibility.A11y_AzureDataStudio_NewJobSteps_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes
